compiler/cpp/src/thrift/generate/t_go_generator.h

```diff
@@ -176,7 +177,8 @@ class t_go_generator : public t_generator {
                                    t_struct* tstruct,
                                    bool is_pointer_field,
                                    bool declare,
-                                   std::string prefix = "");
+                                   std::string prefix = "",
+                                   string alias_name = "");
```

Comment:
can you use `std::string` over `string` here?

it doesn't look like we are very consistent here, but being consistent partially is still better, while the line right above is using `std::string` it's better to keep that.